pages/docs/config/models.mdx

```diff
@@ -1,47 +1,50 @@
 ---
 title: Models
-date: "2021-10-13"
+date: "2022-01-12"
 pullQuote: "Learn how to configure apps using Grouparoo's UI Config."
 ---
 
-Models are how you define the shape you want your records to have in Grouparoo. Once you've created a model, you can add sources and properties to that model to generate different record types. For example, you may want one model that defines leads, another for customers, and another for site visitors. [See this doc](/docs/getting-started/product-concepts) to learn more about Grouparoo's core concepts.
+Models are how you define the shape you want your records to have in Grouparoo.
+Once you've created a model, you can add sources and properties to that model to generate different record types.
+For example, you may want one model that defines leads, another for customers, and another for site visitors.
+See [Product Concepts documentation](/docs/getting-started/product-concepts) to learn more.
 
 With [UI Config](/docs/config) running and registered, follow the steps below to configure your model(s).
 
 ### Step 1: Add a Model
 
-Begin by navigating to the Models page from the navigation bar or setup steps page.
+You can add a new model by clicking "New Model" in the navigation bar or "Add a Model" in the Setup page:
 
 <ImageInBrowserFrame
   centered
   alt="UI Config: Add Model"
   src="/images/docs/guides/config/model--01--add.png"
-  width={2856}
+  width={2389}
   height={1400}
 />
 
-You can add a new model by clicking "Add Model".
+### Step 2: Configure your Model
+
+Give your model a name and select the type.
 
 <ImageInBrowserFrame
   centered
   alt="UI Config: Add Model"
-  src="/images/docs/guides/config/model--02-add.png"
-  width={2864}
-  height={1394}
+  src="/images/docs/guides/config/model--02-name.png"
+  width={2179}
+  height={1396}
 />
 
-Next you can give your model a name. For now, that's all you need to do!
+For now, that's all you need to do!
 
 <ImageInBrowserFrame
   centered
   alt="UI Config: Add Model"
-  src="/images/docs/guides/config/model--03-name.png"
-  width={2872}
+  src="/images/docs/guides/config/model--03-overview.png"
+  width={2192}
   height={1396}
 />
```

Comment:
I don't think this image belongs here?

Reply:
I moved the image to next steps. I just wanted to show what it looked liked after creating the model